contracts/LiquidPledgingBase.sol

```diff
@@ -122,11 +169,26 @@ contract LiquidPledgingBase is Owned {
 
     event GiverUpdated(uint64 indexed idGiver);
 
-    /// @notice Creates a new Delegate
-    function addDelegate(string name, string url, uint64 commitTime, ILiquidPledgingPlugin plugin) returns (uint64 idDelegate) { //TODO return index number
-        require(isValidPlugin(plugin));
+    /// @notice Creates a Delegate Admin with the `msg.sender` as the Admin addr
+    /// @param name The name used to identify the Delegate
+    /// @param url The link to the Delegate's profile often an IPFS hash
+    /// @param commitTime Sets the length of time in seconds the Delegate has to
+    ///   veto when the Delegate delegates to another Delegate and they pledge 
+    ///   those funds to a project
+    /// @param plugin This is Delegate's liquid pledge plugin allowing for 
+    ///  extended functionality
+    /// @return idxDelegate The id number used to reference this Delegate within
+    ///  the admins array
+    function addDelegate(
+        string name,
+        string url,
+        uint64 commitTime,
+        ILiquidPledgingPlugin plugin
+    ) returns (uint64 idxDelegate) { 
+
+        require(isValidPlugin(plugin)); // Plugin check
 
-        idDelegate = uint64(admins.length);
+        idxDelegate = uint64(admins.length);
```

Comment:
why change to `idxDelegate`? I'm pretty sure `idxDelegate` is used to refer to the delegates in a `delegationChain` and `idDelegate` is used to refer to the delegate in the `admins` array

Reply:
Yeah, I don't know. The naming convention makes no sense/is confusing. Ones an ID and ones an index, except they're both indexes.